aptos-move/framework/aptos-framework/sources/Genesis.move

```diff
@@ -62,6 +70,10 @@ module AptosFramework::Genesis {
         epoch_interval: u64,
         minimum_stake: u64,
         maximum_stake: u64,
+        min_lockup_duration_secs: u64,
+        max_lockup_duration_secs: u64,
+        post_genesis_validator_set_change_allowed: bool,
```

Comment:
It uses `allow_new_validators ` below but here still says `post_genesis_validator_set_change_allowed ` and couple other places.

Reply:
Good catch! They're all updated now.